Fix #41: Improve Logger messages for better precision and clarity

AKSHAY-RSOL opened the pull request:
Closes #41
## Changes
- Enhanced all Logger messages to be more descriptive and precise
- Event logs now include title, ID, type, and date
- Gallery logs include filename, category, and file size
- Inventory logs include item name, ID, category, and year
- Blog logs include title and slug
- User logs include email and role information
## Impact
- Improved debugging and monitoring capabilities
- Better audit trail for admin actions
- More actionable log messages

AKSHAY-RSOL committed:
Fix #41: Improve Logger messages for better precision and clarity

- Enhanced event logger messages to include title, ID, type, and date
- Improved gallery logger messages with filename, category, and file size
- Updated inventory logger messages with item name, ID, category, and year
- Enhanced blog logger messages to include title and slug
- Improved user logger messages with email and role information
- Made all log messages more descriptive and actionable for debugging

Mayank447 commented on src/app/api/events/admin/route.ts:
```diff
@@ -203,7 +203,7 @@ export async function PUT(request: NextRequest) {
 
     // Log the action
-    Logger.info("Event updated", {
+    Logger.info(`Updated event: ${updatedEvent?.title} (ID: ${id})`, {
```
Ignore this, as `updatedEvent` is already handled.

Mayank447 reviewed (changes requested):
Please edit the json fields and values for the Logger message instead of putting everything in a line.

AKSHAY-RSOL committed:
fix: restructure Logger messages to use JSON format

- Move all message details from template strings into 'details' object
- Simplify log messages to be more concise
- Maintain all contextual information in structured details field
- Frontend Logger renders based on JSON structure
- Ensures consistent logging format across all API endpoints

satkarjuneja commented:
This issue has been resolved
